### DISCUSSION TITLE
Running EGSnrc within an Apptainer container

### DISCUSSION THREAD

**Discussion by zw-mphys** [nrc-cnrc/EGSnrc#1189] [Questions]
Hello EGSnrc community. I am attempting to run some of my simulations on a cluster by using an Ubuntu Apptainer container.  It seems that my container was created successfully with EGS installed. However, I am having trouble running simulations since EGSnrc requires placement of input files within particular directories (ie. egs_chamber inputs in the egs_chamber directory). I believe this is happening due to containers being read-only. 

To resolve this, is there anyway within EGS to specify alternate directories for input files/outputs? I think this would be a really nice feature of EGS in a future version if this doesn't currently exist.

**Comment by rtownson** [comment]
We have discussed adding this as a future feature, but for now it is required to use the egs_home/egs_chamber directory for all related input files. One alternative is to place a symbolic link to the file in the egs_chamber directory, and keep the actual file elsewhere, but I don't imagine this will solve your issue. Are you able to keep the egs_home directory outside the container?

**Reply by zw-mphys** [reply]
Sym links will likely not work, but placement of egs_home outside the container may be a possibility. I will give this a shot!
